sdk/identity/Azure.Identity/src/CredentialUnavailableException.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Identity
+{
+
+    /// <summary>
+    /// An exception indicating a <see cref="TokenCredential"/> is not available to return an <see cref="AccessToken"/>
```

Comment:
This summary doesn't make it very clear what's the difference between this type and `CredentialUnavailableException`

Reply:
Do you mean the difference between this type and `AuthenticationFailedException`?

Reply:
Yep, copied the wrong thing.